packages/core/src/domain/error/trackRuntimeError.spec.ts

```diff
@@ -15,6 +15,14 @@ describe('runtime error tracker', () => {
   let notifyError: jasmine.Spy
   let stopRuntimeErrorTracking: () => void
 
+  function afterOnErrorInstrumentation(callback: () => void) {
+    const instrumentedOnError = window.onerror!
+    window.onerror = function () {
+      instrumentedOnError.apply(window, arguments as any)
+      callback()
+    }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we already have a spy on `onerror`, maybe just use `onErrorSpy.and.callFake(callback)`? Or you could use `collectAsyncCalls`